### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #3: Update contributing guidelines
Please copy the contributing guidelines from AccessibleWebDev repo and update the links for this repo.

### PR DESCRIPTION
**Pull request by wasup-yash**
Updated Contributing.md file

Motivation for the change:
Closes #3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+Icons - [React Icons Documentation]( https://react-icons.github.io/react-icons)  
+Code Snippets - [React Syntax Highlighter Documentation](https://github.com/react-syntax-highlighter/react-syntax-highlighter)  
+Translations - [next-i18next Documentation](https://next.i18next.com/)  
+Storybook - [Storybook Documentation](https://storybook.js.org/)
+
+## How to contribute
+
+1.  Look through [Pre-existing issues](https://github.com/AccessibleForAll/AccessibleWebDev/issues) or [Raise a new issue](https://github.com/AccessibleForAll/AccessibleWebDev/issues/new/choose) and ask to be assigned. Pull requests made without a corresponding issue will likely be closed.
+2. Please check the [coding standards](https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CODING_STANDARDS.md) page before start contributing.
+3.  [Fork](https://github.com/AccessibleForAll/AccessibleWebDev/fork) the project
```
Comment (EmmaDawsonDev) [inline CONTRIBUTING.md:23]:
```suggestion
3.  [Fork](https://github.com/AccessibleForAll/AccessibleForAll/fork) the project
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+4.  Clone the project:
+    ```bash
+    git clone https://github.com/<your-github-username>/AccessibleWebDev
```
Comment (EmmaDawsonDev) [inline CONTRIBUTING.md:26]:
```suggestion
    git clone https://github.com/<your-github-username>/AccessibleForAll
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1 +1,73 @@
...
+    ```
+5.  Navigate to the project directory:
+    ```bash
+    cd AccessibleWebDev
```
Comment (EmmaDawsonDev) [inline CONTRIBUTING.md:30]:
```suggestion
    cd AccessibleForAll
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1 +1,73 @@
...
+    ```
+6.  Set the upstream repository:
+    ```bash
+    git remote add upstream https://github.com/AccessibleForAll/AccessibleWebDev.git
```
Comment (EmmaDawsonDev) [inline CONTRIBUTING.md:34]:
```suggestion
    git remote add upstream https://github.com/AccessibleForAll/AccessibleForAll.git
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1 +1,73 @@
...
+## Available packages
+
+Icons - [React Icons Documentation]( https://react-icons.github.io/react-icons)  
+Code Snippets - [React Syntax Highlighter Documentation](https://github.com/react-syntax-highlighter/react-syntax-highlighter)  
+Translations - [next-i18next Documentation](https://next.i18next.com/)  
+Storybook - [Storybook Documentation](https://storybook.js.org/)
+
+## How to contribute
+
+1.  Look through [Pre-existing issues](https://github.com/AccessibleForAll/AccessibleWebDev/issues) or [Raise a new issue](https://github.com/AccessibleForAll/AccessibleWebDev/issues/new/choose) and ask to be assigned. Pull requests made without a corresponding issue will likely be closed.
```
Comment (EmmaDawsonDev) [inline CONTRIBUTING.md:21]:
```suggestion
1.  Look through [Pre-existing issues](https://github.com/AccessibleForAll/AccessibleForAll/issues) or [Raise a new issue](https://github.com/AccessibleForAll/AccessibleForAll/issues/new) and ask to be assigned. Pull requests made without a corresponding issue will likely be closed.
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Thanks so much for your help! There's just a couple of links that need updating to match this repo then this should be ready to merge.

Both the issues ended up in the same PR, it's not a big deal with such small PRs but try to remember to make separate branches for each issue next time.

**Comment by wasup-yash** [comment]
> Thanks so much for your help! There's just a couple of links that need updating to match this repo then this should be ready to merge.
> 
> Both the issues ended up in the same PR, it's not a big deal with such small PRs but try to remember to make separate branches for each issue next time.

yes, I will update the corrections and also I also realized they same PR will have 2 changes after i created this one, that's why I did a single commit in the main for `appreciation.yml`PR , I will remember it to branch out first while creating, thanks for the review.

**Comment by EmmaDawsonDev** [comment]
If you want to delete the yaml file from this pr I can merge the other one

**Comment by wasup-yash** [comment]
> If you want to delete the yaml file from this pr I can merge the other one

yes, Deleted `appreciation.yml`.

**Review by EmmaDawsonDev — APPROVED** [review]
Brilliant, thanks!

**Comment by wasup-yash** [comment]
> Brilliant, thanks!

thanks for the support, do I need to look up for some other issues too?

**Comment by EmmaDawsonDev** [comment]
You're more than welcome to look through the current issues or if you find something missing then feel free to create a new issue too. I will hopefully be adding more over time, I just need to start working on a design for this repo so we have something to start with.

**Comment by wasup-yash** [comment]
> You're more than welcome to look through the current issues or if you find something missing then feel free to create a new issue too. I will hopefully be adding more over time, I just need to start working on a design for this repo so we have something to start with.

that's awesome, I would truly like to work on it, looking forward to more contributions.

**Pull request merged**